utility/src/main/java/org/oppia/android/util/locale/OppiaLocale.kt

```diff
@@ -155,6 +155,9 @@ sealed class OppiaLocale {
 
     /** Represents different times of day. */
     enum class TimeOfDay {
+      /** Corresponds to the user's very early morning time. */
+      EARLY_MORNING,
+
       /** Corresponds to the user's morning time. */
       MORNING,
```

Comment:
Perhaps this should be 'MID_MORNING' or 'LATE_MORNING' to contrast with 'EARLY_MORNING'?

Reply:
I went with MID_MORNING, given the range of hours under consideration.